xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/accounts/AccountNftsRequestParams.java

```diff
@@ -0,0 +1,51 @@
+package org.xrpl.xrpl4j.model.client.accounts;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.google.common.primitives.UnsignedInteger;
+import org.immutables.value.Value;
+import org.xrpl.xrpl4j.model.transactions.Address;
+import org.xrpl.xrpl4j.model.transactions.Marker;
+
+import java.util.Optional;
+
+/**
+ * Request parameters for the "account_ntfs" rippled API method.
+ */
+@Value.Immutable
+@JsonSerialize(as = ImmutableAccountNftsRequestParams.class)
+@JsonDeserialize(as = ImmutableAccountNftsRequestParams.class)
+public interface AccountNftsRequestParams {
+
+  /**
+   * Construct a builder for this class.
+   *
+   * @return An {@link ImmutableAccountNftsRequestParams.Builder}.
+   */
+  static ImmutableAccountNftsRequestParams.Builder builder() {
+    return ImmutableAccountNftsRequestParams.builder();
+  }
+
+  /**
+   * The unique {@link Address} for the account.
+   *
+   * @return The unique {@link Address} for the account.
+   */
+  Address account();
+
+  /**
+   * Limit the number of NFTs to retrieve from an account. The server is not required to honor this value.
+   * Must be within the inclusive range 10 to 400.
+   *
+   * @return An optionally-present {@link UnsignedInteger} representing the response limit.
+   */
+  Optional<UnsignedInteger> limit();
```

Comment:
What's the size of the number-type for `limit` in the actual XRP Ledger?

Reply:
`* Must be within the inclusive range 10 to 400.`

Reply:
Cool, thanks!